docs/lib/datastore/fragments/native_common/sync-distributed-data.md

```diff
@@ -0,0 +1,23 @@
+## Distributed data
+
+When working with distributed data it is important to be mindful about the state of the local and the remote systems. DataStore tries to make that as transparent as possible to you; however, some scenarios might require some consideration.

```

Comment:
add comma between "data" and "it"

Reply:
Edit for conciseness:
Be mindful about the local and remote system states when working with distributed data.

Reply:
added